Mirror/Runtime/NetworkTime.cs

```diff
@@ -16,16 +17,21 @@ public static class NetworkTime
 
 
         // Date and time when the application started
-        static readonly DateTime epoch = DateTime.Now;
+        static readonly Stopwatch stopwatch;
```

Comment:
why not stopwatch = new Stopwatch?

Reply:
we can,  but that does not start the stopwatch,   we still need the static constructor